packages/aws-cdk-lib/aws-efs/lib/efs-file-system.ts

```diff
@@ -553,6 +598,26 @@ export class FileSystem extends FileSystemBase {
     if (props.throughputMode === ThroughputMode.ELASTIC && props.performanceMode === PerformanceMode.MAX_IO) {
       throw new Error('ThroughputMode ELASTIC is not supported for file systems with performanceMode MAX_IO');
     }
+
+    const { destinationFileSystem, region, availabilityZone, kmsKey } = props.replicationConfiguration ?? {};
+    if (props.replicationConfiguration) {
+      if (props.replicationOverwriteProtection === ReplicationOverwriteProtection.DISABLED) {
+        throw new Error('Cannot configure `replicationConfiguration` when `replicationOverwriteProtection` is set to `DISABLED`');
```

Comment:
```suggestion
        throw new Error('Cannot configure \'replicationConfiguration\' when \'replicationOverwriteProtection\' is set to \'DISABLED\'');
```
We usually use single quotes for parameter names in error messages.